docs/apps/singularity.md

```diff
@@ -3,6 +3,9 @@
     Here, we describe how to run *NiPreps* with Singularity containers.
     To illustrate the process, we will show the execution of *fMRIPrep*, but these guidelines extend to any other end-user *NiPrep*.
 
+!!! warning "Apptainer"
+    In 2021, Singularity was migrated and became Apptainer. Despite this, all below commands that contain `singularity` will execute Apptainer, and can be replaced with the `apptainer` command with no change in function. 
```

Comment:
```suggestion
    In 2021, [*Singularity* was rebranded as *Apptainer* when the project was transferred to the Linux Foundation](https://apptainer.org/news/community-announcement-20211130/).
    As noted in the community announcement, all the commands below that contain `singularity` as the command line executable will execute *Apptainer* and can be replaced with the `apptainer` command with no change in function. 
```

Reply:
It felt a bit weird not to provide more details. Since there is a community announcement with the name change, linking it seems a good balance between mentioning why and when the name change happened but staying away of explicitly mentioning the trademark conflict that keeping Singularity would have caused (I assume with Sylabs, the company that maintained Singularity before transferring it to Linux).

Reply:
Looks fine to me!